portal/eproms_substudy_tailored_content/src/__tests__/App.test.js

```diff
@@ -0,0 +1,9 @@
+import { mount } from "@vue/test-utils";
+import App from "../components/App.vue";
+
+describe("Mounted App", () => {
+  const wrapper = mount(App);
+  test("has the expected html structure", () => {
```

Comment:
Nice! Are there automated tests we can run now? Can you add some short notes how to run them if so?

Reply:
Ivan, no, it is not currently automated yet, but I did add note the README that details which command to execute to run the test.

Reply:
Thank you! I'll plan on adding those commands to CI when I refactor it next